.github/workflows/ansible-test-integration.yml

```diff
@@ -45,92 +45,13 @@ jobs:
-            --diff --python 3.8 --docker --coverage digital_ocean_firewall
-
-      - name: Run the digital_ocean_floating_ip integration test
-        run: |
-          ansible-test integration -v --color --retry-on-error --continue-on-error \
-            --diff --python 3.8 --docker --coverage digital_ocean_floating_ip
-
-      - name: Run the digital_ocean_image_info integration test
-        run: |
-          ansible-test integration -v --color --retry-on-error --continue-on-error \
-            --diff --python 3.8 --docker --coverage digital_ocean_image_info
-
-      - name: Run the digital_ocean_load_balancer_info integration test
-        run: |
-          ansible-test integration -v --color --retry-on-error --continue-on-error \
-            --diff --python 3.8 --docker --coverage digital_ocean_load_balancer_info
-
-      - name: Run the digital_ocean_region_info integration test
-        run: |
-          ansible-test integration -v --color --retry-on-error --continue-on-error \
-            --diff --python 3.8 --docker --coverage digital_ocean_region_info
-
-      - name: Run the digital_ocean_size_info integration test
-        run: |
-          ansible-test integration -v --color --retry-on-error --continue-on-error \
-            --diff --python 3.8 --docker --coverage digital_ocean_size_info
-
-      - name: Run the digital_ocean_snapshot_info integration test
-        run: |
-          ansible-test integration -v --color --retry-on-error --continue-on-error \
-            --diff --python 3.8 --docker --coverage digital_ocean_snapshot_info
-
-      - name: Run the digital_ocean_sshkey integration test
-        run: |
-          ansible-test integration -v --color --retry-on-error --continue-on-error \
-            --diff --python 3.8 --docker --coverage digital_ocean_sshkey
-
-      - name: Run the digital_ocean_tag integration test
-        run: |
-          ansible-test integration -v --color --retry-on-error --continue-on-error \
-            --diff --python 3.8 --docker --coverage digital_ocean_tag
-
-      - name: Run the digital_ocean_volume_info integration test
-        run: |
-          ansible-test integration -v --color --retry-on-error --continue-on-error \
-            --diff --python 3.8 --docker --coverage digital_ocean_volume_info
+      - name: Run the integration tests
+        run: ansible-test integration --docker -v --color --retry-on-error --continue-on-error --python 3.8 --diff --coverage
 
       # ansible-test support producing code coverage date
       - name: Generate coverage report
-        run: |
-          mkdir -p tests/output/coverage
-          ansible-test coverage xml -v --requirements --group-by command --group-by version
-        working-directory: ansible_collections/community/digitalocean
+        run: ansible-test coverage xml -v --requirements --group-by command --group-by version
+        working-directory: ./ansible_collections/${{env.NAMESPACE}}/${{env.COLLECTION_NAME}}
```

Comment:
Curious why we have to set the `working-directory` here when it's set in `defaults.run.working-directory`?

Reply:
Ah, hadn't noticed you'd set this. Should likely roll that into https://github.com/ansible-collections/collection_template/blob/main/.github/workflows/ansible-test.yml